bool classifier_running

string classifier_name
string classifier_state
string classifier_connected

string target_class
bool target_detected

string pantilt_device
string pantilt_connected
bool has_position_feedback
float32 min_area_ratio
float32 scan_speed_ratio
float32 scan_tilt_offset
float32 scan_pan_angle
float32 track_speed_ratio
float32 track_tilt_offset

bool is_running
bool is_scanning
bool is_tracking


